### ISSUE TITLE
missing libs

### ISSUE THREAD

**Issue by gaveer** [ChrisKader/LFMPlus#12]
```
2x LFM+\core.lua:5: Cannot find a library instance of "LibUIDropDownMenu-4.0".
[string "=[C]:"]: in function 'error'
[string "@Bugger\Libs\LibStub\LibStub.lua"]:23: in function 'GetLibrary'
[string "@LFM+\core.lua"]:5: in main chunk
```

**Comment by gaveer** [comment]
@ChrisKader

**Comment by ChrisKader** [comment]
Thank you @gaveer

I missed the fact that LibUIDropDownMenu is inside of a folder within a folder. I will be uploading a fixed package shortly.

**Comment by ChrisKader** [comment]
This should now be fixed with release [v1.4.8](https://github.com/ChrisKader/LFMPlus/releases/tag/v1.4.8)
